Core/src/org/sleuthkit/autopsy/casemodule/CaseNodeData.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2011-2017 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.casemodule;
+
+import java.nio.BufferUnderflowException;
+import java.nio.ByteBuffer;
+
+/**
+ * An object that converts case data for a case directory coordination service
+ * node to and from byte arrays.
+ */
+public final class CaseNodeData {
+
+    private static final int CURRENT_VERSION = 0;
+    
+    private int version;
+    private boolean errorsOccurred;
```

Comment:
Now that this data is available to the MultiUserCaseManager, it should be be read in MultiUserCaseManager.getCases and stored in MultiUserCaseManager.MultiUserCase.  As stated in the DoD of the story for this work in JIRA:

"The Status column of the open multi-user case dialog is populated using the flag in the node data instead of the presence or absence of an "alert" file in the case directory."

Looking at the code in MultiUserCase.

`   
     CaseStatus getStatus() {
            if(caseDirectoryPath.resolve("autoingest.alert").toFile().exists()) {
                return CaseStatus.ALERT;
            } else {
                return CaseStatus.OK;
            }
        }
`

this code, as is, will fail to report errors except for cases created when alert files were still written.

Also, considering our concerns about how the multi-user case panel code is performing, we should check for legacy alert files in MultiUserCaseManager.getCases, where we have already done a file listing.